MeetupPlatformApi/DataTransferObjects/MeetupInputDto.cs

```diff
@@ -1,13 +1,25 @@
-﻿namespace MeetupPlatformApi.DataTransferObjects;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.DataTransferObjects;
 
 public class MeetupInputDto
 {
+    [Required]
+    [MaxLength(30)]
+    [DataType(DataType.Text)]
+    [Display(Name = "Name of meetup")]
     public string Name { get; set; }
 
+    [Required]
+    [DataType(DataType.DateTime)]
     public DateTime StartTime { get; set; }
 
-    public DateTime EndTime { get; set; }
+    [DataType(DataType.DateTime)]
+    public DateTime? EndTime { get; set; }
```

Comment:
Why did you change the field type and why is it not required now?

Reply:
because theoretically there may be such a situation that we know the start time of the rally, but we don’t know what time it will end, for example, if at the end we arrange an afterparty

Reply:
1. Meetup's schedules are usually quite strict
2. It's a major business-rule change and it's not in the scope of this task